src/general/other.css

```diff
@@ -30,22 +42,32 @@
 }
 
 /* Revert new Nitro icon by setting item */
-.premiumTabItem-1QTfBr .icon-Zc-uZZ {
-  display: none;
+[d="M1.84006 5.4367C2.19536 5.4367 2.49139 5.14651 2.49139 4.79823C2.49139 4.44996 2.19536 4.15977 1.84006 4.15977H1.48476C1.12947 4.15977 0.83329 4.44996 0.83329 4.79823C0.83329 5.14651 1.12947 5.4367 1.48476 5.4367H1.84006ZM15.282 13.1569C18.6574 11.6477 20.0785 7.81669 18.539 4.6241C17.4731 2.36029 15.2229 1.02527 12.8542 0.909088H6.16283C5.57062 0.909088 5.15606 1.37355 5.15606 1.89597C5.15606 2.47634 5.62989 2.88271 6.16283 2.88271H7.88006C8.23535 2.88271 8.53139 3.17289 8.53139 3.52117C8.53139 3.86945 8.23535 4.15977 7.88006 4.15977H3.67575C3.32046 4.15977 3.02428 4.44996 3.02428 4.79823C3.02428 5.14651 3.32046 5.4367 3.67575 5.4367H6.69578C7.05108 5.4367 7.34711 5.72702 7.34711 6.0753C7.34711 6.42358 7.05108 6.71376 6.69578 6.71376H4.80077C4.44547 6.71376 4.14943 7.00395 4.14943 7.35222C4.14943 7.7005 4.44547 7.99083 4.80077 7.99083H6.10357C6.16283 8.68738 6.34048 9.38394 6.63652 10.0224C8.11697 13.215 12.0252 14.6081 15.282 13.1569ZM9.22163 8.83321C8.36181 6.99779 9.18283 4.82663 11.0552 3.98379C12.9277 3.14096 15.1426 3.94576 16.0024 5.78119C16.8622 7.61661 16.0413 9.78777 14.1688 10.6306C12.2964 11.4734 10.0814 10.6686 9.22163 8.83321Z"] {
```

Comment:
Please do not do this. Use classes or nth child within parent. (Same for both.)

Reply:
There is unfortunately no class name for the logo so this is the only way to hide it.

Reply:
...just do `premiumLabel-3HPvdB > svg`
![image](https://cdn.discordapp.com/emojis/849741528565678100.png?size=48&quality=lossless)

Reply:
> Please do not do this. Use classes or nth child within parent. (Same for both.)

Fixed it

Reply:
> ...just do `premiumLabel-3HPvdB > svg`, or, hell, to future-proof it, `div[class*="premiumLabel-"] > svg` ![image](https://camo.githubusercontent.com/fe874d0a86eb5e86ffb2597f8fe00ab93e78334af73e3bcb4e6f7e7be40973b4/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3834393734313532383536353637383130302e706e673f73697a653d3438267175616c6974793d6c6f73736c657373)

Thanks

Reply:
You didn't fix it. Get rid of the `d[...]` block.

Just do this:
```css
div[class*="premiumLabel-"] > svg {
 display: none;
}
```

Reply:
> You didn't fix it. Get rid of the `d[...]` block.
> 
> Just do this:
> 
> ```css
> div[class*="premiumLabel-"] > svg {
>  display: none;
> }
> ```

Done